src/com/example/ppj05/Task02.java

```diff
@@ -0,0 +1,37 @@
+package com.example.ppj05;
+/*Write a program  which reads four integer numbers and prints the difference between
+the largest and the smallest of them. Don’t use arrays, strings or collections. */
+import java.util.Scanner;
+
+public class Task02 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        int num1 = scanner.nextInt();
+        int num2 = scanner.nextInt();
+        int num3 = scanner.nextInt();
+        int num4 = scanner.nextInt();
+
+        int max = Math.max(Math.max(num1, num2), Math.max(num3, num4));
```

Comment:
We should not use `Math` utills methods in this task.  This task needed for practice `if else` constructions